packages/fxa-content-server/app/scripts/lib/nimbus/index.ts

```diff
@@ -31,9 +32,17 @@ export default class Nimbus {
 
       this.experiments = await resp.json();
     } catch (err) {
+      if (err.name === 'TimeoutError') {
```

Comment:
Seems reasonable. One question though, if the backend service is down or misconfigured, I'd assume, we'd also get a timeout. In this case, is there some secondary monitoring that'd tip us off that something is wrong? 

If so, doing this seems fine. If not this could mask a real problem.

Reply:
That seems okay. Let's roll with this. I just noticed the auth server proxies the call, and seems like it does report the sentry error, so should be good to go.